.circleci/config.yml

```diff
@@ -37,6 +37,7 @@ commands:
             else
               python -m venv venv
               . venv/bin/activate
+              pip install --upgrade pip
```

Comment:
I was having an issue with grpcio randomly being built from source rather than wheels, particularly on python 3.6. I think this fixed it? And we might as well make sure we work against recent pips...